updates.html

```diff
@@ -3,4 +3,19 @@
 title: Updates
 layout: default
 ---
-<h1>Updates</h1>
\ No newline at end of file
+<main class="ds-base ds-l-container">
+  <div class="ds-l-row ds-u-fill--primary-darker">
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--display ds-u-color--white">
+      Updates
+    </div>
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--h1 ds-u-color--white">
+      Keep up with the latest news from BCDA
+    </div>
+  </div>
+  <div class="ds-l-col--12 ds-u-padding-y--2">
+    {% include updates/intro.html %}
+  </div>
+  <div class="ds-u-padding-y--2">
+    {% include updates/2020-10-28.html %}
```

Comment:
Is it possible to have a wildcard used so we can include all content inside the `updates/` directory?